ui/tests/integration/helpers/date-format-test.js

```diff
@@ -1,27 +1,37 @@
 import { module, test } from 'qunit';
 import { setupRenderingTest } from 'ember-qunit';
-import { dateFormat } from '../../../helpers/date-format';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
 
 module('Integration | Helper | date-format', function(hooks) {
   setupRenderingTest(hooks);
 
-  test('it is able to format a date object', function(assert) {
+  test('it is able to format a date object', async function(assert) {
     let today = new Date();
-    let result = dateFormat([today, 'YYYY']);
-    assert.ok(typeof result === 'string');
-    assert.ok(result !== 'Invalid Date', 'it is not an invalid date');
-    assert.ok(Number(result) >= 2017);
+    this.set('today', today);
+
+    await render(hbs`<p data-test-date-format>Date: {{date-format today "YYYY"}}</p>`);
+    assert
+      .dom('[data-test-date-format]')
+      .includesText(today.getFullYear(), 'it renders the date in the year format');
   });
 
-  test('it supports date timestamps', function(assert) {
-    let today = new Date().getTime();
-    let result = dateFormat([today, 'YYYY']);
-    assert.ok(Number(result) >= 2017);
+  test('it formats the date as specified', async function(assert) {
+    let today = new Date();
```

Comment:
i think you'll want to keep the same value we were using before in this test -- `Date().getTime()`, since that is testing that the helper can handle various types of inputs. it's not testing the various types of outputs, if that makes sense.

Reply:
Makes sense.  Amended.